src/lando/dockerflow/views.py

```diff
@@ -0,0 +1,84 @@
+from django.db import connection
+from django.db.utils import OperationalError
+from django.http import JsonResponse
+from django.views import View
+
+from lando.dockerflow.decorators import disable_caching, log_request
+
+
+class DockerflowView(View):
+    """
+    This is the base class view for all Dockerflow related views.
+
+    It handles common functionality needed by all Dockerflow views.
+    """
+
+    @disable_caching
+    @log_request
+    def dispatch(self, request, *args, **kwargs):
+        return super().dispatch(request, *args, **kwargs)
+
+    @staticmethod
+    def _json_response(data, status):
+        return JsonResponse(data, status=status, json_dumps_params={"indent": 2})
+
+
+class VersionView(DockerflowView):
+    """
+    This view handles the version information of Lando.
+
+    It returns a JSON response containing the version information.
+    """
+
+    def get(self, request):
+        try:
+            from lando.version import version
+
+            data = {
+                "version": version,
+            }
+            status = 200
+        except ImportError:
+            data = {"error": "Service Unavailable"}
+            status = 503
```

Comment:
The try section of try/except blocks should contain as little code as possible (see [PEP 8](https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations)).
```python
try:                                                                                                                                                                                                                  
    from lando.version import version
except ImportError:
    data = {"error": "Service Unavailable"}
    status = 503 
else:
    data = { 
        "version": version,
    }   
    status = 200 
...
```

Reply:
Done.